docs/products/access/topics/oidc/overview/scopes-and-claims.md

```diff
@@ -44,32 +44,47 @@ identify their source and also prevent from potential clashes/collisions.
 ## Optional claims
 
 ### ACR
->**Note:**
-> This feature requires the usage of the Onegini CIM as identity provider.
 
-Via ACR (Authentication Context Class Reference) you request that a specific authentication context must be met upon successful authentication.
+Via ACR (Authentication Context Class Reference) you request that a specific authentication context must be met upon successful
+authentication.
+
+#### CIM
+
+Only supported ACR values will be accepted when using the Onegini CIM as the identity provider.
+CIM accepts ACR values format containing authentication level and optionally identification level.
+
+Example values:
+
+| ACR value                                                          | Description                                                                                                |
+|--------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| urn:onegini.com:oidc:authentication_level:2                        | Requires an authentication level of at least 2 from Onegini CIM.                                           |
+| urn:onegini.com:oidc:authentication_level:3:identification_level:2 | Requires an authentication level of at least 3 and an identification level of at least 2 from Onegini CIM. |
+
+All supported values are exposed via the [Discovery API](../../../api-reference/oidc/discovery.md).
 
-The table below summarizes currently supported values. The available values are also exposed via the [Discovery API](../../../api-reference/oidc/discovery.md).
+For CIM IdP, only a single ACR value can be specified. Sending multiple values will result in a `Bad Request` error.
 
-| ACR value                                   | Description                                                                      |
-|---------------------------------------------|----------------------------------------------------------------------------------|
-| urn:onegini.com:oidc:authentication_level:1 | Requires an authentication level of at least 1 from Onegini CIM.  |
-| urn:onegini.com:oidc:authentication_level:2 | Requires an authentication level of at least 2 from Onegini CIM.  |
-| urn:onegini.com:oidc:authentication_level:3 | Requires an authentication level of at least 3 from Onegini CIM.  |
-| urn:onegini.com:oidc:authentication_level:4 | Requires an authentication level of at least 4 from Onegini CIM.  |
+#### OAuth
+
+For OAuth-based identity providers (e.g. Tulip, Identity Broker), requested ACR values are forwarded upstream to the configured identity
+provider.
+
+Supported ACR values from IdP OpenID configuration are exposed via the [Discovery API](../../../api-reference/oidc/discovery.md). However,
+Access does not validate provided ACR values and allows those not in OpenID configuration.
```

Comment:
This is not completely clear to me. Be more clear on what we expect from the external IDP and what Access does. Also we may need to be explicit that not all IdP types support all ACR values listed in the discovery endpoints.

Reply:
Please check if it's clear now